app/viewmodels/SecurityStatementsViewModel.scala

```diff
@@ -16,9 +16,259 @@
 
 package viewmodels
 
-import models.SecurityStatementsForEori
+import config.AppConfig
+import models.FileFormat.{Csv, Pdf}
+import models.FileRole.SecurityStatement
+import models.{SecurityStatementFile, SecurityStatementsByPeriod, SecurityStatementsForEori}
+import play.api.i18n.Messages
+import play.twirl.api.{Html, HtmlFormat}
+import utils.Utils.{
+  ddComponent, divComponent, dlComponent, dtComponent, emptyString,
+  h1Component, h2Component, linkComponent, pComponent, spanComponent
+}
+import views.helpers.Formatters.{dateAsDayMonthAndYear, dateAsMonthAndYear}
+import views.html.components.{h2Inner, h3Inner, missing_documents_guidance, pInner, requestedStatements}
+
+import java.time.LocalDate
+
+case class SecurityStatementsViewModel(pageTitle: Option[String],
+                                       backLink: Option[String],
+                                       header: HtmlFormat.Appendable,
+                                       requestedStatementNotification: HtmlFormat.Appendable,
+                                       currentStatements: HtmlFormat.Appendable,
+                                       missingGuidance: HtmlFormat.Appendable,
+                                       statementServiceParagraph: HtmlFormat.Appendable,
+                                       requestStatementsLink: HtmlFormat.Appendable,
+                                       details: SecurityStatementsViewModelDetails)
```

Comment:
Could you please advise where this is being used? Seems we are using this only in Spec and in population via SecurityStatementsViewModel's apply method. If it is not being used then we can remove this field.

Reply:
This has been addressed in the commit [73dfe78](https://github.com/hmrc/customs-financials-documents-frontend/pull/101/commits/73dfe781532fd8bddb0967c9eeb15974440beaf7)